dkg/exchanger.go

```diff
@@ -131,7 +131,7 @@ func newExchanger(tcpNode host.Host, peerIdx int, peers []peer.ID, vals int, sig
 // signatures of the group according to public key of each DV.
 func (e *exchanger) exchange(ctx context.Context, sigType sigType, set core.ParSignedDataSet) (map[core.PubKey][]core.ParSignedData, error) {
 	// Start the process by storing current peer's ParSignedDataSet
-	duty := core.NewSignatureDuty(int64(sigType))
+	duty := core.NewSignatureDuty(uint64(sigType))
```

Comment:
Would change sigType to uint64 instead.

Reply:
I don't that's required, what'd be the reason to change type for `sigType` which is currently an `int` to `uint64`?

slots, committee indices etc should be uint64 because the spec mandates that.